pyaedt/modeler/Model3DLayout.py

```diff
@@ -105,6 +105,86 @@ def obounding_box(self):
         """ """
         return self.oeditor.GetModelBoundingBox()
 
+
+    @aedt_exception_handler
+    def colinear_heal(self, selection, tolerance=0.1):
+        """Remove small edges of selected object list.
+
+        Parameters
+        ----------
+        selection : list or str
+            list of primitives to heal
+        tolerance :  float
+            tolerance
+
+        Returns
+        -------
+        bool
+
+        Examples
+        --------
+        >>> from pyaedt import Hfss3dLayout
+        >>> h3d=Hfss3dLayout(specified_version="2021.1")
+        >>> h3d.modeler.layers.add_layer("TOP")
+        >>> l1=h3d.modeler.primitives.create_line("TOP", [[0,0],[100,0]],  0.5, name="poly_1")
+        >>> l2=h3d.modeler.primitives.create_line("TOP", [[100,0],[120,-35]],  0.5, name="poly_2")
+        >>> h3d.modeler.unite([l1,l2])
+        >>> h3d.modeler.colinear_heal("poly_2", 0.25)
+        True
+        """
+        if isinstance(selection, str):
+            selection = [selection]
+        self.oeditor.Heal(["NAME:Repair", "Selection:=", selection, "Type:=", "Colinear", "Tol:=",
+                           self.primitives.arg_with_dim(tolerance)])
+        return True
+
+
+
+    @aedt_exception_handler
+    def expand(self, object_to_expand,  size=1, expand_type="ROUND", replace_original=False):
+        """Expand the object by a specific size. If replace_original is ``False`` the method will create a new object
+
```

Comment:
```suggestion
        """Expand the object by a specific size.

        If replace_original is ``False`` the method will create a new object.

```
Short description should always be one sentence.  Longer description should be added below.